API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
```

Comment:
If I understand this model correctly, an Activity is made up of several Lesson Plans. If that's the case, I might suggest we use the term "Modules" instead of "Activities."